src/v2/components/TopBar/components/PrimarySearch/index.js

```diff
@@ -106,6 +106,7 @@ export default class PrimarySearch extends PureComponent {
         {mode === 'resting' && <HomeLink />}
 
         <SearchInput
+          globallyFocusOnKey="/"
```

Comment:
Added this setting, which should allow us to be able to add additional commands later to nested search inputs (like in channels).